library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -410,7 +410,7 @@
     <string name="action_disconnect">Disconnect</string>
     <string name="action_play">Play</string>
     <string name="action_dismiss">Dismiss</string>
-    <string name="action_reset">Reset</string>
+    <string name="action_reset">Proceed to Reset</string>
```

Comment:
I'd rather create a new string, this one might be used somewhere else?

Reply:
I already checked that it is not used anywhere other than reset fragment. Should we still make a new string?